runtime-sdk/src/module.rs

```diff
@@ -304,10 +304,16 @@ pub trait AuthHandler {
         // Default implementation doesn't do anything.
         Ok(())
     }
+
+    /// Perform any action after call, within the transaction context.
```

Comment:
Document what happens if this returns an error (transaction fails and updates are rolled back).